diff --git a/def/defines.h b/def/defines.h
index 39d0658..fb4998d 100644
--- a/def/defines.h
+++ b/def/defines.h
@@ -27,9 +27,9 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
 #define DEFINES_H
 
 // OS constants
-#define APP_WIN						true
+#define APP_WIN						false
 #define APP_MAC						false
-#define APP_X11						false
+#define APP_X11						true
 #define APP_PORTABLE				false //at least one of the 3 OS must be true too!
 
 // Languages
@@ -45,7 +45,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
 #define APP_NAME_INTERN				"TIPP10"
 #define APP_NAME 					"TIPP10"
 #define APP_URL 					"http://www.tipp10.com"
-#define APP_DB 						"tipp10v2.template"
+#define APP_DB 						"../data/tipp10v2.template"
 #define APP_USER_DB					"tipp10v2.db"
 
 // Update constants
